server/test/controllers/MockMessageApi.scala

```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import play.api.i18n.{Lang, Messages, MessagesApi}
+import play.api.mvc
+import play.api.mvc.Result
+import play.mvc.Http.RequestHeader
+
+class MockMessageApi extends MessagesApi {
```

Comment:
it depends, I recommend to use scalamock first, otherwise than refactor, and finally use mockito